mongo/options/encryptoptions.go

```diff
@@ -105,7 +106,14 @@ func (ro *RangeOptions) SetMax(max bson.RawValue) *RangeOptions {
 // SetSparsity sets the range index sparsity.
 // Beta: The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes.
 func (ro *RangeOptions) SetSparsity(sparsity int64) *RangeOptions {
-	ro.Sparsity = sparsity
+	ro.Sparsity = &sparsity
+	return ro
+}
+
+// SetTrimFactor sets the range index trim factor.
```

Comment:
Optional: Add a blank line here to make the "Beta" warning more visible.

```suggestion
// SetTrimFactor sets the range index trim factor.
//
```